core/templates/dev/head/profile/signup.html

```diff
@@ -80,62 +81,50 @@ <h2 class="oppia-signup-page-title">Complete Your Registration</h2>
 
         <hr>
 
-        <p style="font-size: 1em;">
-          The {{SITE_NAME}} site is an open commons of learning resources, and all material
-          on it is freely reusable and shareable.
+        <p style="font-size: 1em;" translate translate-value-sitename="{{SITE_NAME}}">
```

Comment:
Sorry, don't understand (and I think your previous comment is missing the name of the tag). Does translate-values work now?